src/components/storage/CockpitStorageIntegration.jsx

```diff
@@ -137,9 +137,13 @@ const CockpitStorageConfirmationModal = ({ handleCancelOpenModal, handleConfirmO
         >
             <TextContent>
                 <Text>
-                    {_("The advanced storage editor lets you resize, delete, and create partitions. It can set up LVM and much more.")}
+                    {_("The storage editor lets you resize, delete, and create partitions. " +
```

Comment:
The indentation is not nice ;(

Please adjust:
``` diff
                 <Text>
                     {_("The storage editor lets you resize, delete, and create partitions. " +
-        "It can set up LVM and much more. " +
-        "It is meant to be used as an advanced utility and not intended to be used in a typical installation.")}
+                       "It can set up LVM and much more. " +
+                       "It is meant to be used as an advanced utility and not intended to be used in a typical installation.")}
                 </Text>
```

Also please use prefix: storage in the commit message, and make the screenshot with english language. :)